main/src/ca/uwaterloo/flix/language/errors/NameError.scala

```diff
@@ -214,7 +207,7 @@ object NameError {
     * @param loc  the location of the suspicious type variable.
     */
   case class SuspiciousTypeVarName(name: String, loc: SourceLocation) extends NameError {
-    def summary: String = s"Suspicious type variable. Did you mean: '${name.capitalize}'?"
+    def summary: String = s"Suspicious type variable '$name'. Did you mean: '${name.capitalize}'?"
```

Comment:
`int` is in the list of suspicious names, but `Int` is not a valid type anymore (or at least we are headed in that direction).

So we may just want to take two parameters here, one for the suspicious name and one for our guess.